src/libshadertrap/src/executor.cc

```diff
@@ -1201,41 +1201,142 @@ bool Executor::CheckEqualBuffers(CommandAssertEqual* assert_equal) {
     return false;
   }
 
+  bool result = true;
+  size_t start_index = 0;
+  std::vector<CommandAssertEqual::FormatEntry>& format_entries = assert_equal->GetFormatEntries();
+  const Token t = assert_equal->GetStartToken();
```

Comment:
Try to avoid single-letter variable names such as `t` - they don't help the reader.